help/availability.md

```diff
@@ -0,0 +1,26 @@
+Availability of submissions and announcement order
+==================================================
+
+Submissions to arXiv are typically made public Sunday through
+Thursday, with no announcements Friday or Saturday. Check the [current
+time](https://arxiv.org/localtime) at the arXiv main site, with the
+amount of time remaining before the deadline. Submissions are made public as part of the scheduled announcement process. This includes new submissions as well as [replacements](/help/replace), [withdrawal notices](/help/withdraw), [cross listings](/help/cross) and [journal reference](/help/jref). Journal reference submissions do not appear in the mailings, but are still processed at that time. 
+
+The final [arXiv identifier](arxiv_identifier) cannot be assigned until the paper is announced, so please do not ask for it ahead of announcement. It is not possible to make exceptions, nor accelerate the announcement schedule regardless of your circumstance. 
+
+It may be the case that a submission appears in a different identifier month due to various factors. **Note:** The arXiv identifier cannot be back-dated, and identifiers are ordered based on the final timestamp of submitter action (i.e. when "submit" is clicked). 
+
+
+| Submissions received between<br />(all times Eastern US) | Will be announced<br />(all times Eastern US) | Mailed to subscribers |
+|:---:|:---:|:---:|
+| Monday 14:00 &ndash; Tuesday 14:00 | Tuesday 20:00 | Tuesday night / Wednesday morning |
+| Tuesday 14:00 &ndash; Wednesday 14:00 | Wednesday 20:00 | Wednesday night / Thursday morning | 
+| Wednesday 14:00 &ndash; Thursday 14:00 | Thursday 20:00 | Thursday night / Friday morning |
+| Thursday 14:00 &ndash; Friday 14:00 | Sunday 20:00 | Sunday night / Monday morning |
+| Friday 14:00 &ndash; Monday 14:00 | Monday 20:00 | Monday night / Tuesday morning |
+
+<a name="holiday"></a>
```

Comment:
I like the idea of referring to holidays, but am hesitant to have another place to post that info. 
Can we refer people instead to see https://status.arxiv.org/ for planned and unplanned events that may affect announcements?

Reply:
I think status is good for aggregation tools, but we _always_ get questions about days of announcement. Having it in 3 places is fine if we're going to maintain all of them, but it's useful to have a one-stop shop with a link. While linking and announcing through the blog is probably fastest, maybe we need to have a spot where users can quickly find all announcement-related info together?

Reply:
It might make a lot of sense to surface the contents of the Holidays.pm module at a place like arxiv.org/holidays.

Reply:
I instead changed the wording on the page to link to the status and blog locations. I think the best thing to do will be to add some Pendo code to this page when we do have a deferred mailing, but to not update it otherwise.